reana-ui/src/pages/status/Status.js

```diff
@@ -20,6 +20,13 @@ import { healthMapping } from "~/util";
 
 import styles from "./Status.module.scss";
 
+const statusColorMapping = {
+  available: "mediumseagreen",
+  running: "green",
```

Comment:
Thanks for nice comments. I changed the `available` color to be a bit lighter, and `running` to be a bit more darker than `mediumseagreen` (used this tool: https://www.color-hex.com/color/3cb371).

Before:
![Screenshot from 2021-12-08 10-25-07](https://user-images.githubusercontent.com/4990025/145186141-eb57a3d0-02e6-454d-8c5a-8562f53542e4.png)
Now:
![Screenshot from 2021-12-08 10-40-49](https://user-images.githubusercontent.com/4990025/145186173-6db27ee2-58c3-4519-be12-375794e4efa0.png)

Let me know what you think. Also cc @tiborsimko

Reply:
Looks much better IMO!

Reply:
Cosmetic comment: On my setup, perhaps a bit "less pale" green would look a tiny bit nicer. We don't have to stress the the difference so much?

Functional comment: One issue I'm seeing is the "nodes available" stat displayed in pale green. Actually, the nodes are busy running k8s, and if 2 workflows are running, surely some of the nodes are running these workfow pods and job pods as well. So I'd suggest to simply use "darker green" to express the notion of "stuff is running". I think it would play better with the "dark green" being used to show running things in the other plots in the picture.

(We don't have to go into distinguishing which nodes are running pods and which don't; they would be an unnecessary overkill. In any way, the nodes _are_ busy running the basic k8s ifrastructure, which is the purpose of that plot.)